src/release/breaking-changes/notifications.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Removing Notification.visitAncestor
+description: Notifications only traverse parents that are notification
+listeners.
+---
+
+## Summary
+
+Notifications are more efficient by traversing only parents that
+are notification listeners.
+
+## Context
+
+The notification API traversed the element tree in order to locate a
+notification receiver. This led to some unfortunate performance
+characteristics:
+
+  * If there was no receiver for a given notification type, the entire element
+    tree above the notification would be traversed and type checked.
```

Comment:
nit: notification -> notification dispatch point or some such since notifications are not part of the tree.

Reply:
done